src/DI/WebpackExtension.php

```diff
@@ -73,9 +79,7 @@ public function loadConfiguration()
 				new Statement(Client::class)
 			]);
 
-		$assetResolver = $builder->addDefinition($this->prefix('assetResolver'))
-			->setClass(AssetResolver\AssetResolverInterface::class)
-			->setFactory($config['assetResolver']);
+		$assetResolver = $this->setupAssetResolver($config);
 
 		if ($this->debugMode && $config['debugger']) {
```

Comment:
why there is needed to have $this->debugMode && $config[debugger], shouldnt this be one thing?